sdk/cosmos/tests.yml

```diff
@@ -17,51 +17,28 @@ jobs:
       AdditionalModules:
         - name: azure-cosmos-benchmark
           groupId: com.azure
-        - name: azure-cosmos-examples
-          groupId: com.azure
-      TimeoutInMinutes: 90
+      TimeoutInMinutes: 120
```

Comment:
This number is going in the wrong direction. Do we have plans to try and increase the performance of these test runs to get the overall time down?

Reply:
Yes, we are working on that. I have started the work with @danieljurek on Dynamic Resource Provisioning. 
No need to block this PR for that, as that change will come after this gets merged in.

Reply:
@weshaggard - The reason I increased some of the timeouts is because I merged some of the CIs to reduce the number of CIs. 
Earlier these were running in 1 hour, 25 mins, so added another 30 mins to the timeout value as a buffer.